src/api/indexedDb.ts

```diff
@@ -103,45 +161,37 @@ export class IndexedDB {
 	}
 
 	private async initDb() {
+		const dbVersion = localStorageWorker.getDBVersion();
```

Comment:
no we don't need to store indexeddb version, we get current user db version inside upgrade